### DISCUSSION TITLE
不知道是artalk还是模板的问题

### DISCUSSION THREAD

**Discussion by penndu** [ArtalkJS/Artalk#131] [Q&A] [answered]
通过邮件或者控制中心无法定位评论，如图：

https://bu.dusays.com/2022/04/26/6267fdab40586.gif

**Comment by penndu** [comment]
详细描述一下，通过控制中心跳转评论，会出现空白屏，需向上移动页面才会显示出评论，但评论先之后，没有定位评论。

**Comment by qwqcode** [comment]
你在 侧边栏控制中心 - 站点，添加了你的网站域名吗？

**Reply by penndu** [reply]
> 你在 侧边栏控制中心 - 站点，添加了你的网站域名吗？

加了！

**Comment by qwqcode** [comment]
![F5FD203B-0871-42F6-9C5A-750BD1D57369](https://user-images.githubusercontent.com/22412567/165358790-dd6de892-f932-48f3-8104-59bcf2157143.png)

**Reply by penndu** [reply]
> ![image](https://user-images.githubusercontent.com/46226528/165358899-79f21619-6b19-48f5-9e49-8b0b5893879d.png)

**Reply by qwqcode** [reply]
有点奇怪，哈哈 先睡觉了

**Reply by penndu** [reply]
> 有点奇怪，哈哈 先睡觉了

早点休息，晚安！

**Reply by qwqcode** [reply]
刚刚试了下，大概估计，你的主题，可能检测 URL 的 hash 跳转到对应的锚点，主题代码获取 scrollTop，因 Artalk 动态加载后，变化的滚动条高度，获取有点问题。我猜可以搞个 setTimeOut 解决？

**Reply by penndu** [reply]
> 刚刚试了下，大概估计，你的主题，可能检测 URL 的 hash 跳转到对应的锚点，主题代码获取 scrollTop，因 Artalk 动态加载后，变化的滚动条高度，获取有点问题。我猜可以搞个 setTimeOut 解决？

赶紧去休息吧，有时间再慢慢研究，我可以先在控制中心回~~~

**Reply by qwqcode** [reply]
![image](https://user-images.githubusercontent.com/22412567/165442315-8841a707-8fb6-4f4b-89d3-5f432a855c4e.png)

也许是你的主题这段代码，和 Artalk 冲突了

**Reply by penndu** [reply]
> 刚刚试了下，大概估计，你的主题，可能检测 URL 的 hash 跳转到对应的锚点，主题代码获取 scrollTop，因 Artalk 动态加载后，变化的滚动条高度，获取有点问题。我猜可以搞个 setTimeOut 解决？

经测试关闭 scrollreveal 功能可解决该问题。

**Reply by penndu** [reply]
> 也许是你的主题这段代码，和 Artalk 冲突了

参考柚子回复 https://github.com/volantis-x/hexo-theme-volantis/discussions/746#discussioncomment-2643415

**Reply by qwqcode** [reply]
你可以判断一下，阻止这段代码的执行： 

```js
if (location.hash.indexOf('#atk-comment-') === 0) { return; }
```

**Reply by qwqcode** [reply]
我没有测试过，只是猜测，哈哈
具体修复方法还要看实际情况

**Comment by penndu** [comment, marked as answer]
去掉评论区域的滚动动画，修改 [comments/index.ejs](https://github.com/volantis-x/hexo-theme-volantis/blob/dev/layout/_plugins/comments/index.ejs#L9) 文件，将 reveal 删去。

目前已经临时修复完成~
